aws/resource_aws_elb_test.go

```diff
@@ -1195,15 +1157,18 @@ func testAccCheckAWSELBExists(n string, res *elb.LoadBalancerDescription) resour
 }
 
 const testAccAWSELBConfig = `
+data "aws_availability_zones" "available" {
+  state = "available"
+}
+
 resource "aws_elb" "bar" {
-  availability_zones = ["us-west-2a", "us-west-2b", "us-west-2c"]
+  availability_zones = ["${data.aws_availability_zones.available.names[0]}", "${data.aws_availability_zones.available.names[1]}", "${data.aws_availability_zones.available.names[2]}"]
 
   listener {
     instance_port = 8000
     instance_protocol = "http"
     lb_port = 80
-    // Protocol should be case insensitive
-    lb_protocol = "HttP"
```

Comment:
This additional change fixes the testing on master to support Terraform 0.12, where the configuration value is propagated to the Terraform state:

```
--- FAIL: TestAccAWSELB_basic (206.80s)
    testing.go:568: Step 0 error: Check failed: Check 9/10 error: aws_elb.bar: Attribute 'listener.206423021.lb_protocol' expected "http", got "HttP"
```